cmd/kubeadm/app/phases/addons/dns/dns.go

```diff
@@ -132,6 +132,15 @@ func coreDNSAddon(cfg *kubeadmapi.ClusterConfiguration, client clientset.Interfa
 		return errors.Wrap(err, "error when parsing CoreDNS service template")
 	}
 
+	if printManifest {
+		fmt.Printf("%s\n", coreDNSDeploymentBytes)
+		fmt.Println("---")
+		fmt.Printf("%s\n", coreDNSConfigMapBytes)
+		fmt.Println("---")
+		fmt.Printf("%s\n", coreDNSServiceBytes)
+		return nil
```

Comment:
we should start with initial `---` on the first line:

```suggestion
		fmt.Println("---")
		fmt.Printf("%s\n", coreDNSDeploymentBytes)
		fmt.Println("---")
		fmt.Printf("%s\n", coreDNSConfigMapBytes)
		fmt.Println("---")
		fmt.Printf("%s\n", coreDNSServiceBytes)
		return nil
```